assets/js/components/adminbar/AdminBarWidgets.js

```diff
@@ -28,31 +28,45 @@ import Data from 'googlesitekit-data';
 import AdminBarImpressions from './AdminBarImpressions';
 import AdminBarClicks from './AdminBarClicks';
 import AdminBarUniqueVisitors from './AdminBarUniqueVisitors';
+import AdminBarUniqueVisitorsGA4 from './AdminBarUniqueVisitorsGA4';
 import AdminBarSessions from './AdminBarSessions';
+import AdminBarSessionsGA4 from './AdminBarSessionsGA4';
 import AdminBarActivateAnalyticsCTA from './AdminBarActivateAnalyticsCTA';
 import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
 import { Row, Cell } from '../../material-components';
 import { withWidgetComponentProps } from '../../googlesitekit/widgets/util/get-widget-component-props';
+import { useFeature } from '../../hooks/useFeature';
 const { useSelect } = Data;
 
 // Widget slugs.
 const WIDGET_IMPRESSIONS = 'adminBarImpressions';
 const WIDGET_CLICKS = 'adminBarClicks';
 const WIDGET_VISITORS = 'adminBarUniqueVisitors';
 const WIDGET_SESSIONS = 'adminBarSessions';
+
 // Search Console widgets.
 const AdminBarImpressionsWidget =
 	withWidgetComponentProps( WIDGET_IMPRESSIONS )( AdminBarImpressions );
 const AdminBarClicksWidget =
 	withWidgetComponentProps( WIDGET_CLICKS )( AdminBarClicks );
+
 // Analytics Widgets.
 const AdminBarUniqueVisitorsWidget = withWidgetComponentProps(
 	WIDGET_VISITORS
 )( AdminBarUniqueVisitors );
 const AdminBarSessionsWidget =
 	withWidgetComponentProps( WIDGET_SESSIONS )( AdminBarSessions );
```

Comment:
I realise it's a divergence from the IB, but we could simplify this change and also make it more efficient by taking the following approach instead, and removing the rendering changes. I'd suggest we go with this, what do you think?

```suggestion
const ga4ReportingEnabled = isFeatureEnabled( 'ga4Reporting' );

// Analytics Widgets.
const AdminBarUniqueVisitorsWidget = withWidgetComponentProps(
	WIDGET_VISITORS
)( ga4ReportingEnabled ? AdminBarUniqueVisitors : AdminBarUniqueVisitorsGA4 );

const AdminBarSessionsWidget = withWidgetComponentProps( WIDGET_SESSIONS )(
	ga4ReportingEnabled ? AdminBarSessions : AdminBarSessionsGA4
);
```

Reply:
Thanks, @techanvil, I thought the same; however, I found this method was used in our codebase. I did so to keep it consistent.
https://github.com/google/site-kit-wp/blob/3c36bc8f555ae98dcc5ac21595503661572eed88/assets/js/components/wp-dashboard/WPDashboardWidgets.js#L61-L78

Reply:
Let me know WDYT?

Reply:
I see - thanks for pointing that out. In that case, sure, let's keep it as you've implemented it for the sake of consistency, considering as well that it does follow the IB. In the long run we'll be removing the `ga4Reporting` flag anyway and this code will be simplified then.